### PR TITLE
Amap and PVS Studio reports in CI/CD

### PR DESCRIPTION
**Pull request by drunkbatya**
# What's new

- added .map file analyse in CI/CD
- added PVS Studio static code analyse report in CI/CD
- added clang-format to toolchain

# Verification 

- there should be a separate .map report in mariadb and pvs studio report on update server for each new commit to PR
- lint fbt targets like `lint` and `format` should work without installed clang-format

# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Comment by hedger** [comment]
Pass PVS-Studio credentials through GH secrets. See https://pvs-studio.com/en/docs/manual/6579/

**Comment by drunkbatya** [comment]
> Pass PVS-Studio credentials through GH secrets. See https://pvs-studio.com/en/docs/manual/6579/

This is not necessary cause Puppet provides our runners with all the essential stuff including PVS credentials

File: scripts/amap_mariadb_insert.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+import mariadb
+import sys
+import os
+
+if len(sys.argv) < 7:
```
Comment (hedger) [inline scripts/amap_mariadb_insert.py]: using argparse module would be better

**Review by hedger — DISMISSED** [review]

**Comment by skotopes** [comment]
> > Pass PVS-Studio credentials through GH secrets. See https://pvs-studio.com/en/docs/manual/6579/
> 
> This is not necessary cause Puppet provides our runners with all the essential stuff including PVS credentials

We've been talking about it: please use standard way.

**Comment by drunkbatya** [comment]
> > > Pass PVS-Studio credentials through GH secrets. See https://pvs-studio.com/en/docs/manual/6579/
> > 
> > 
> > This is not necessary cause Puppet provides our runners with all the essential stuff including PVS credentials
> 
> We've been talking about it: please use standard way.

Fixed

File: scripts/amap_mariadb_insert.py (hunk 1 above)
Comment (drunkbatya) [reply scripts/amap_mariadb_insert.py]: Thanks, fixed

**Review by hedger — DISMISSED** [review]

**Review by hedger — DISMISSED** [review]

**Review by skotopes — DISMISSED** [review]

**Review by skotopes — APPROVED** [review]

**Pull request merged**